pkg/storage/mock/storage.go

```diff
@@ -0,0 +1,152 @@
+package mock
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"time"
+
+	"github.com/Azure/brigade/pkg/brigade"
+)
+
+var (
+	// Now is the date used in all stub date fields.
+	Now = time.Now()
```

Comment:
Curious, are time-based race conditions a problem with go tests? In Ruby/JS specs it's common to bind "now" to a known value to deal with freezing/jumping time (e.g. https://github.com/travisjeffery/timecop). Just a wonder, not blocking this PR whatsoever.

Reply:
Not to the extent it was in rails.